lib/jira/base.rb

```diff
@@ -428,8 +428,8 @@ def url
         end
       end
       if @attrs['self']
-        the_url = @attrs['self'].sub(@client.options[:site],'')
-        the_url = "/#{the_url}" if (the_url =~ /^\//).nil?
```

Comment:
I'm not sure I can safely remove this.
No test are failing but it was suppose to fix an issue 2 years ago: https://github.com/sumoheavy/jira-ruby/commit/f37244eeb12b5408a4ca829ca25338dbb83aeac4

Reply:
It does break `save` if the `site` includes a trailing slash:
given a self attribute of 
`https://myacc.atlassian.net/rest/api/2/issue/55642`
and a `site` of`
`https://myacc.atlassian.net/`
you get back
`rest/api/2/issue/55642`
that's passed to `save(attrs, path = url)` as the default for the second argument
and from there it goes directly into the request, which expects paths to start with a `/`

Reply:
Nice catch, would you be ready to submit a PR to fix this bug?

Reply:
I should be able to in the next couple of days.

Reply:
For now I've worked around by stripping the slash off our `site` argument